packages/xrpl/src/sugar/getFeeXrp.ts

```diff
@@ -43,3 +43,22 @@ export default async function getFeeXrp(
   // Round fee to 6 decimal places
   return new BigNumber(fee.toFixed(NUM_DECIMAL_PLACES)).toString(BASE_10)
 }
+
+/**
```

Comment:
Shouldn't this be a part of `autofill` instead of a separate method?

Reply:
Originally I was under the impression the `tx_blob` submitted to the fee rpc needed to be signed. Signing it in the autofill created a circular dependency. It does not need to be signed, only encoded. So I could move this into the `autofill`. However, I believe this does replace the current `Fee` request so if the current fee request is a sugar, then this should also be a sugar as well.

I can update the set fee in auto fill with this new estimate fee functionally.

Reply:
We shouldn't update the current functionality, because this new feature is not live yet (it's part of the Hooks amendment, right?), and we want this library to be useable on mainnet right now.